docs/examples/Power Flow Example.ipynb

```diff
@@ -1139,16 +1139,16 @@
      "output_type": "stream",
      "text": [
       "Node data with invalid results\n",
-      "[[0.99940117 0.99268579 0.99452137]\n",
-      " [0.99934769 0.98622639 0.98935286]\n",
-      " [0.99928838 0.97965401 0.98409554]\n",
-      " [0.         0.         0.        ]\n",
-      " [0.99915138 0.96614948 0.97329879]\n",
-      " [0.99907317 0.95920586 0.96775071]\n",
-      " [0.9989881  0.95212621 0.96209647]\n",
-      " [0.         0.         0.        ]\n",
-      " [0.99879613 0.93753005 0.95044796]\n",
-      " [0.9986885  0.92999747 0.94444167]]\n",
+      "[[ 9.99401170e-001  9.92685785e-001  9.94521366e-001]\n",
+      " [ 9.99347687e-001  9.86226389e-001  9.89352855e-001]\n",
+      " [ 9.99288384e-001  9.79654011e-001  9.84095542e-001]\n",
+      " [ 1.05085968e+056 -4.32115156e-239 -5.52267234e+127]\n",
```

Comment:
maybe caused by python3.13? well... the data was invalid either way so it should be fine